inventory/templates/inventory/catalogue_for_type.html

```diff
@@ -61,39 +58,34 @@ <h1>Item catalogue for {{ item_type }}</h1>
                     </div>
 
                 </td>
-                {% if can_change_items %}
-                    <td>
-                        <a href="{% url "inventory:catalogue_update_item" type_id=item_type item_id=item.id %}"
-                           class="btn btn-sm btn-primary">
-                            Edit
-                        </a>
-                    </td>
-                {% endif %}
-                {% if can_maintain_ownerships %}
-                    <td>
+                <td>
+                    {% if can_maintain_ownerships %}
                         <div class="d-flex justify-content-between">
                             <div>
                                 {% for owner in item.currently_in_possession %}
```

Comment:
Owners listed in the table only include owners of active items, whereas the filter option also includes owners of inactive items. While it is nice to include inactive owners, this is not indicated anywhere in the UI. Perhaps also include inactive owners in the table and make active owners bold?